categorized mechants update

BALIST0N opened the pull request:
Every trader sell different things now
- Prapor : ammo, silencers, magazines, grenades and the GL
- Therapist : meds, food, drinks, stimulants, maps, keys & keycards
- Fence : barter, quests items, special items
- Skier : rails, mounts, flashlights & lasers, knifes
- Peacekeeper : nightvisions, money, optics & scopes
- Mechanic : all vital weapons mods (pistol grips, handguards, barrels ), stocks, muzzles, foregrips, iron sights
- Ragman : all gear, secure cases, containers & cases
- ItemTrader : all secret and hidden items of the game, ammo boxes, naked weapons
- Preset Trader  : remain the same, unchanged

BALIST0N committed:
categorized mechants update

every trader sell different things now
- Prapor : ammo, silencers, magazines, grenades and the GL
- Therapist : meds, food, drinks, stimulants, maps, keys & keycards
- Fence : barter, quests items, special items
- Skier : rails, mounts, flashlights & lasers, knifes
- Peacekeeper : nightvisions, money, optics & scopes
- Mechanic : all vital weapons mods (pistol grips, handguards, barrels ), stocks, muzzles, foregrips, iron sights
- Ragman : all gear, secure cases, containers & cases
- ItemTrader : all secret and hidden items of the game, ammoboxes,naked weapons
- Preset Trader  : remain the same, unchanged

BALIST0N committed:
No error for Flea market

BUT GOD HOW THIS THING WORK HELP ME THIS IS GETTING MY NERVES

polivilas commented:
Great idea, but I feel that all item trader should still exist. Could you add it as a different trader? Thanks!

BALIST0N committed:
Added "everythingTrader" Part1

a trader who sell every everything

BALIST0N committed:
added "everythingTrader" part 1 

"itemTrader" renamed secretTrader

BALIST0N committed:
Delete ItemTrader.json

BALIST0N committed:
Added "Everything Trader" Part 2 

blablabla

BALIST0N committed:
Delete ItemTrader.json

BALIST0N committed:
Added "Everything trader" & "SecretTrader"

- everythingTrader sells everything as requested 
- Secret trader is just a rename (who sell secret items)

ghost commented:
Sure ! I've just updated it, you can check it

I am also working on the flea market to make this working so there a file added (ragfair/search.json) and a case catch in "HandlingRequest" function
it just create the left menu categories and avoid error but it is still useless for the moment

polivilas commented:
:+1:
